File: Assignment1.java
import java.util.Scanner;
import java.lang.Math;

public class Assignment1 {

    public static final Scanner input = new Scanner(System.in);

    public static void main(String[] args) {

       long startTimeODD = System.nanoTime();
       OddNums();
       long endTimeODD = System.nanoTime();
       long totalTimeODD = endTimeODD - startTimeODD;
       System.out.println("OddNums method took: " + totalTimeODD + " nanoseconds");

       long startTimeCAL = System.nanoTime();
       CalenderOps();
       long endTimeCAL = System.nanoTime();
       long totalTimeCAL = endTimeCAL - startTimeCAL;
       System.out.println("CalenderOps took: " + totalTimeCAL + " nanoseconds");

       long startTimeMAX = System.nanoTime();
       FindMax();
       long endTimeMAX = System.nanoTime();
       long totalTimeMAX = endTimeMAX - startTimeMAX;
       System.out.println("FindMax took: " + totalTimeMAX + " nanoseconds");

       long startTimeTEMP = System.nanoTime();
       ConvertTemp();
       long endtimeTEMP = System.nanoTime();
       long totalTimeTEMP = endtimeTEMP - startTimeTEMP;
       System.out.println("ConverTemp took: " + totalTimeTEMP + " nanoseconds");
        

    }

    public static void OddNums(){
        System.out.println("__________________________");
        System.out.println("Print Odd Nums:");
        System.out.println("The Odd Numbers are:");
		for (int i = 1; i <= 1001; i++) {
			if (i % 2 != 0) {
				System.out.print(i+ ((i%20==9) ? "\n" : " "));
			}
        }
        System.out.println();
    }

    public static void CalenderOps(){
        System.out.println("__________________________");
        System.out.println("Calender Conversions:");
        System.out.println("Enter number of days");
        double days = input.nextInt();

        double years = days/365;
        double months = days/12;

        System.out.println("Days: " + days);
        System.out.println("Years: " + years);
        System.out.println("Months: " + months);

        System.out.println();
    }

    public static void FindMax(){
        System.out.println("__________________________");
        System.out.println("Find Max:");
        System.out.println("Enter 3 integers: ");
        int num1 = input.nextInt();
        int num2 = input.nextInt();
        int num3 = input.nextInt();
        
        int max = Math.max(num1,num2);
        if(max>num2){
            max = Math.max(num1,num3);    
       }
        else { 
            max = Math.max(num2,num3);    
     }
        System.out.println("The max integer is: " + max);
        System.out.println();
    }

    public static void ConvertTemp(){
        System.out.println("__________________________");
        System.out.println("Temperature Conversion:");
        System.out.println("Enter your degrees in Fahrenheit");
        double tempF = input.nextDouble();

        double tempC = (tempF-32.0)*(5.0/9.0);

        System.out.println("Temp (F) to Temp (C): ");
        System.out.printf("%.2f", tempC);
        //System.out.printf("Temp (F) to Temp (C): " + temp);
        System.out.println();

    }
}

File: Assignment1.cpp
#include <iostream>
#include <stdlib.h>
#include <math.h>
#include <iomanip>
#include <chrono>

using namespace std;


void OddNums(){

    cout << "Print Odd Numbers from 1 to 10001";
    cout << "\n\n";
    for (int a=1; a<=10001; a+=2) {
     
        cout << " " << a << " ";\
        if ((a % 20) == 9) // print 10 lines per line
        cout << endl;
    }

    cout << endl;

}

void CalenderOps(){

    cout << "__________________________" << endl;
    cout << "Calender Conversions:";
    cout << endl;

    cout << "Enter how many days:" << endl;
    double day;
    cin >> day; 
    cout << endl;

    double year = day/365.0;
    double month = day * 0.032855; 

    cout << "Days: " << day << endl;
    cout << "Years: " << year << endl;
    cout << "Months: " << month << endl;

}

void FindMax() {
    cout << "__________________________" << endl;
    cout << "Find Max:";
    cout << endl;

    cout << "Enter 3 integers: " << endl;
    double num1;
    double num2;
    double num3;

    cin >> num1 >> num2 >> num3;
    cout << endl;

	int max = num1; /* assume x is the largest */

	if (num2 > max) { /* if y is larger than max, assign y to max */
		max = num2;
	} /* end if */

	if (num3 > max) { /* if z is larger than max, assign z to max */
		max = num3;
	} /* end if */

	cout << "The max integer is: " << max << endl;

}

void ConvertTemp(){
    cout << "__________________________" << endl;
    cout << "Temperature Conversion: ";
    cout << endl;

    cout << "Enter your temperature in Fahrenheit" << endl;
    
    double tempF;
    cin >> tempF;
    cout << endl;

    double tempC = (tempF-32.0)*(5.0/9.0);

    cout << "Temp (F) to Temp (C): ";
    cout << fixed << setprecision(2) << tempC << endl; 

}

//main has to be called last??
int main() {
    //std::cout << "Main was reached";

    auto startODD = chrono::steady_clock::now();
    OddNums();
    auto endODD = chrono::steady_clock::now();
    auto totalODD = endODD - startODD;
    cout << "OddNums function took: " << chrono::duration <double, nano> (totalODD).count() << " nanoseconds" << endl;

    auto startCAL = chrono::steady_clock::now();
    CalenderOps();
    auto endCAL = chrono::steady_clock::now();
    auto totalCAL = endCAL - startCAL;
    cout << "CalenderOps function took: " << chrono::duration <double, nano> (totalCAL).count() << " nanoseconds" << endl;

    auto startMAX = chrono::steady_clock::now();
    FindMax();
    auto endMAX = chrono::steady_clock::now();
    auto totalMAX = endMAX - startMAX;
    cout << "FindMax function took: " << chrono::duration <double, nano> (totalMAX).count() << " nanoseconds" << endl;

    auto startTEMP = chrono::steady_clock::now();
    ConvertTemp();
    auto endTEMP = chrono::steady_clock::now();
    auto totalTEMP = endTEMP - startTEMP;
    cout << "CalenderTemp function took: " << chrono::duration <double, nano> (totalTEMP).count() << " nanoseconds" << endl;
}
